common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -210,6 +215,7 @@ def go_to_sequential_position(self, position):
 
 
 @attr('a11y')
+@override_waffle_flag(COURSEWARE_USE_LEGACY_FRONTEND, active=True)
```

Comment:
@doctoryes I don't fully understand how bok choy tests work, but I suspect that the override_waffle_flag I added here won't actually work, since this Python is not calling LMS code directly but is instead operating on a webpage served by LMS.